docs/solr.txt

```diff
@@ -36,102 +45,135 @@ index.search.solr.zookeeper-url=localhost:2181
 
 A number of additional configuration options pertaining to the creation of new collections (which is only supported in SolrCloud operation mode) can be configured to control sharding behavior among other things. Refer to the <<config-ref>> for a complete listing of those options.
 
+SolrCloud leverages Zookeeper to coordinate collection and configset information between the Solr servers. The use of Zookeeper with SolrCloud provides the opportunity to significantly reduce the amount of manual configuration required to use Solr as a back end index for JanusGraph.
 
-===== Connecting via HTTP
 
-When connecting to Solr via HTTP by setting the `mode` equal to `http` a single or list of URLs for the Solr instances must be provided.
+==== Configset Configuration
 
-[source, properties]
-----
-index.search.backend=solr
-index.search.solr.mode=http
-index.search.solr.http-urls=http://localhost:8983/solr
-----
+A configset is required to create a collection. The configset is stored in Zookeeper to enable access to it across the Solr servers.
 
-Additional configuration options for controlling the maximum number of connections, connection timeout and transmission compression are available for the HTTP mode. Refer to the <<config-ref>> for a complete listing of those options.
+* Each collection can provide its own configset when it is created, so that each collection may have a different configuration. With this approach, each collection must be created manually.
```

Comment:
Isn't it only the configset that *must* be created manually? JanusGraph will create the collection automatically if it doesn't exist (at least for SolrCloud) though as was discussed in the recent issue it'll do so by assuming the corresponding configset has the same name.

Reply:
You're right. I like creating the collection manually because it's basically the same as creating a core in standalone mode. Maybe it would be more clear if I remove the Zookeeper verification steps in the Using an individual configset section?

Reply:
@pluradj It would be cleaner but also it's useful information I wouldn't want to be lost. Maybe move the Zookeeper verification steps to a troubleshooting or separate section? Regardless it's an improvement over what was there.

Reply:
Actually, I had already duplicated it in the troubleshooting https://pluradj.github.io/janusgraph.org/latest/solr.html#_troubleshooting_2